---
layout: archive
title: "Publications"
permalink: /publications/
author_profile: true
---


{% if author.googlescholar %}
  You can also find my articles on <u><a href="{{author.googlescholar}}">my Google Scholar profile</a>.</u>
{% endif %}

{% include base_path %}

<!--
{% for post in site.publications reversed %}
  {% include archive-single.html %}
{% endfor %}
-->

<!--为了简单下面自己进行自动 
<p style="color:blue;">You can change the text color of a whole sentence or paragraph...</p>
<p>...or you can change the text color of one <span style="color:green;">word</span> or even a single l<span style="color:red;">e</span>tter.</p>
-->

Peer-reviewed Publications
==========================
<sup>\#</sup> co-first author, $^*$ corresponding author. $^\dagger$ Idea development, $^\ddagger$ software contribution, <sup>&para;</sup> Statistics contribution.

<style>
/* unvisited link */
a:link {
  color: blue;
}

/* visited link */
a:visited {
  color: black;
}

/* mouse over link */
a:hover {
  color: hotpink;
}

/* selected link */
a:active {
  color: blue;
}
</style>

<ol reversed>

<li>
Jie Ren<sup>(#)</sup>, Ningning Zhang<sup>(#)</sup>, <b>Xiangjie Li</b><sup>(&para;)</sup>, Xiaogang Sun, Jinagping Song <sup>(*)</sup>.(2021). Identification of reference genes for gene expression studies among different developmental stages of murine hearts. BMC Developmental Biology.21:13. <a href="https://doi.org/10.1186/s12861-021-00244-6"> https://doi.org/10.1186/s12861-021-00244-6 </a>.
</li>

<li>
Hang Hu,  Zhong Li<sup>(*) </sup>,<b>Xiangjie Li</b>$^{(\ddagger)}$,Minzhe Yu adn Xiutao Pan.(2021) 
ScCAEs: deep clustering of single-cell RNA-seq via convolutional autoencoder embedding and soft K-means. Breifings in Bioinformatics, bbab321,<a href="https://doi.org/10.1093/bib/bbab321"> https://doi.org/10.1093/bib/bbab321 </a>
</li>

<li>
Justin Lakkis, David Wang, Yuanchao Zhang, Gang Hu, Kui Wang, Huize Pan, Lyle Ungar, Muredach P. Reilly, <b>Xiangjie Li</b><sup>(*) </sup>, Mingyao Li<sup>(*) </sup>. A joint deep learning model for simultaneous batch effect correction, denoising and clustering in single-cell transcriptomics. Published online in Genome Research.  <a href="https://genome.cshlp.org/content/early/2021/05/25/gr.271874.120.abstract"> https://genome.cshlp.org/content/early/2021/05/25/gr.271874.120.abstract </a>.
</li>

<li>
Jian Hu, <b>Xiangjie Li</b> $^{(\dagger)}$, Kyle Coleman, Amelia Schroeder, David J.Irwin, Edward B. Lee, Russell T Shinohara1, Mingyao Li<sup>(*) </sup>. Integrating gene expression, spatial location and histology to identify spatial domains and spatially variable genes by graph convolutional network. Accepted by Nature Methods. <a href="https://www.biorxiv.org/content/10.1101/2020.11.30.405118v1"> https://www.biorxiv.org/content/10.1101/2020.11.30.405118v1 </a>.
</li>

<li> 
<b> Xiangjie Li </b>, Kui Wang, Yafei Lyu, Jingxiao Zhang, Dwight Stambolian,Katalin Susztak, Muredach P. Reilly, Gang Hu <sup> (*) </sup> and Mingyao Li <sup>(*) </sup>. (2020). Deep learning enables accurate clustering and batch effect removal in single-cell RNA-seq. Nature Communications 11, 2338 (2020). <a href="https://eleozzr.github.io/desc">https://eleozzr.github.io/desc </a>.
</li>

<li>

Liang Chen<sup>(#)</sup>, <b>Xiangjie Li </b> <sup>(#) </sup>, Mingquan Chen, Yi Feng and Chenglong Xiong<sup>(*)</sup>. (2020). The ACE2 expression in human heart indicates new potential mechanism of heart injury among patients infected with SARS-CoV-2. Cardiovascular Research. 116(6): 1097–1100.<span style="color:red;">(ESI Hot Papers and ESI Highly Cited Papers, cited 659 times)</span>
</li>

<li>
Mengmeng Li<sup>(#)</sup>, Liang chen<sup>(#)</sup>, Jingxiao Zhang, Chenglong Xiong<sup>(*) </sup> and <b> Xiangjie Li</b><sup>(*) </sup>. (2020) The SARS-CoV-2 receptor ACE2 expression of maternal-fetal interface and fetal organs by single-cell transcriptome study. Plos One. 15(4):e0230295. <span style="color:red;">(ESI Highly Cited Papers, cited 240 times)</span>

</li>

<li>
Jie Ren<sup>(#)</sup>, Liang Chen<sup>(#)</sup>, Ningning Zhang, Xiao Chen, Qian Zhao, Kai Chen, <b><b>Xiangjie Li</b></b><sup>(&para;)</sup>, Frank Ruschitzka, Firat Duru<sup>(*)</sup> and Jiangping Song<sup>(*)</sup>. (2020). Plasma testosterone and arrhythmic events in male patients with arrhythmogenic right ventricular cardiomyopathy. ESC Heart Failure. 7(4): 1547–1559.
</li>

<li>
Jian Hu, <b>Xiangjie Li</b>, Gang Hu, Yafei Lyu, Katalin Susztak and Mingyao Li<sup>(*)</sup>.(2020). Iterative transfer learning with neural network for clustering and cell type classification in single-cell RNA-seq analysis. Nature Machine Intelligence. <a href="https://doi.org/10.1038/s42256-020-00233-7"> https://doi.org/10.1038/s42256-020-00233-7</a>,
</li>

<li>
Rou Zhong, <b>Xiangjie Li</b>$^{(\dagger)}$, Jingixao Zhang<sup>(*)</sup>. (2021). Model-free robust conditional independence feature screening.Journal of Applied Statistics and Management (In Chinese). <a href="https://doi.org/10.13860/j.cnki.sltj.20200619-002"> https://doi.org/10.13860/j.cnki.sltj.20200619-002 </a>.

</li>

<li>
Xinyi Xu, <b>Xiangjie Li</b>$^{(\dagger)}$, Jingxiao Zhang<sup>(*)</sup>. (2020). Regularization methods for high-dimensional sparse control function models. Journal of Statistical Planning and Inference. 206:111-126
</li>

<li>
Shishi Liu, <b>Xiangjie Li</b>$^{(\dagger)}$, Jingixao Zhang<sup>(*)</sup>.(2020). Ultrahigh dimensional feature screening for additive model with multivariate response. Journal of Statistical Computation and Simulation. 90(5): 775-799.
</li>

<li>
<b>Xiangjie Li</b>, Jingxiao Zhang<sup>(*)</sup>. (2020). Sufficient dimension folding via tensor inverse regression. Journal of Statistical Computation and Simulation. 90(8):1413-1429.
</li>

<li>
<b>Xiangjie Li</b>, Xuejun Ma, Jingxiao Zhang<sup>(*)</sup>. (2018). Conditional quantile correlation screening procedure for ultrahigh-dimensional varying coefficient models. Journal of Statistical Planning and Inference. 197:69-92.
</li>

<li>
Xinyi Xu, <b>Xiangjie Li</b>$^{(\dagger)}$, Jingixiao Zhang<sup>(*)</sup>.(2018). Sufficient dimension reduction and prediction through cumulative slicing PFC. Journal of Statistical Computation and Simulation.88(6):1172-1190.
</li>


<li>
<b>Xiangjie Li</b>, Yanyan Wu, Jingxiao Zhang <sup>(*)</sup>. (2018). Robust dimension reduction method based on sliced inverse regression. Statistical Research (In Chinese), 5(7): 115-124.
</li>

<li>
<b>Xiangjie Li</b>, Jingxiao Zhang<sup>(*)</sup>. (2018). Robust rank conditional feature screening for ultrahigh dimensional data. Statistics & Information Forum (In Chinese), 33(4): 6-12.
</li>

<li>
<b>Xiangjie Li</b>, Xuejun Ma and Jingxiao Zhang<sup>(*)</sup>. (2017). Robust feature screening for varying coefficient models via quantile partial correlation. Metrika, 80(1): 17-49.
</li>


<li>
<b>Xiangjie Li</b>, Lei Wang and Jingxiao Zhang<sup>(*)</sup>. (2017). A model-free feature screening for ultrahigh-dimensional data based on kernel density estimation. Journal of Statistical Computation and Simulation. 87(12): 2450-2468.
</li>

<li>
Jingxiao Zhang<sup>(*,#)</sup>, <b>Xiangjie Li</b><sup>(#)</sup>, Haiming Guo. (2016). Robust variable screening for ultrahigh dimensional. Statistics & Information Forum (In Chinese), 32(4): 9-12.
</li>

</ol>



Papers in preparation
===================

<ol reversed>

<li>
Zishu Zhan,  <b>Xiangjie Li</b><sup>(*)</sup>, Jingxiao Zhang<sup>(*)</sup>. Projective resampling imputation estimation method for missing covariates problems. <i> Under review </i>
</li>


<li>
Xinyi Xu, <b>Xiangjie Li</b><sup>(*)</sup>. Batch effect for scRNA-seq gene expression using propensity scoring matching. <i> In preparation </i>
</li>

<li>
Jie Ren<sup>(#)</sup>, <b>Xiangjie Li</b><sup>(#)</sup>, et al. scRNA-seq revealed the postnatally developmental trajectories in human TGA myocardial tissue.<i> Under review </i> 
</li>


</ol>
